ui/app/templates/components/administration-subnav.hbs

```diff
@@ -0,0 +1,17 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<div class="tabs is-subnav" {{did-insert this.keyboard.registerNav type="subnav"}} {{will-destroy this.keyboard.unregisterSubnav}}>
+  <ul>
+    <li><LinkTo @route="administration.index" @activeClass="is-active">Overview</LinkTo></li>
+    <li><LinkTo @route="administration.tokens" @activeClass="is-active">Tokens</LinkTo></li>
+    <li><LinkTo @route="administration.roles" @activeClass="is-active">Roles</LinkTo></li>
+    <li><LinkTo @route="administration.policies" @activeClass="is-active">Policies</LinkTo></li>
+    <li><LinkTo @route="administration.namespaces" @activeClass="is-active">Namespaces</LinkTo></li>
+    {{#if (can "list sentinel-policy")}}
```

Comment:
This conditional is about whether the ENT feature is available, rather than about the ACL policy?

Reply:
Yep! We let you load the Administration section as long as you can list roles, policies, tokens and namespaces (in other words: you are a management token) but it should still be a usable section without Sentinel Policies (ent).

So we don't show sentinel policies in subnav/index page, nor attempt to preload them from the parent route, if the Sentinel Policies feature (ent) is missing.

Reply:
We also route-guard for other "is management" abilities in the [parent route](https://github.com/hashicorp/nomad/blob/8bfe03a835d108d4b2245b1da67a3eb5557cf8ce/ui/app/routes/administration.js#L20-L29)